File: BackEnd/postman/Tratamiento.py
import requests
def postear(data, url):
    return requests.post('http://127.0.0.1/syme/BackEnd/services/'+url+'.php', data=data).content

url='Tratamiento'
data={
  'accion':'crear',
  'dosis': None,
  'idTipoTratamiento': None,
  'intervaloTiempo': None,
  'fechaInicio': None,
  'idViaSuministro': None,
  'duracionTratamiento': None,
}
print postear(data,url)
print

url='Tratamiento'
data={
  'accion':'agregarTipoTratamiento',
  'tipoTratamiento': None,
}
print postear(data,url)
print

url='Tratamiento'
data={
  'accion':'listarPorPaciente',
  'idPaciente': None,
}
print postear(data,url)
print

url='Tratamiento'
data={
  'accion':'listarTipoTratamiento',
}
print postear(data,url)
print

url='Tratamiento'
data={
  'accion':'actualizarReceta',
  'idTratamiento': None,
  'idConsulta': None,
  'idMedico': None,
}
print postear(data,url)
print

url='Tratamiento'
data={
  'accion':'agregarViaSuministro',
  'viaSuministro': None,
}
print postear(data,url)
print

url='Tratamiento'
data={
  'accion':'listarViaSuministro',
}
print postear(data,url)
print

url='Tratamiento'
data={
  'accion':'recetar',
  'idTratamiento': None,
  'idConsulta': None,
  'idMedico': None,
}
print postear(data,url)
print

url='Tratamiento'
data={
  'accion':'actualizarViaSuministro',
  'viaSuministro': None,
  'idViaSuministro': None,
}
print postear(data,url)
print

url='Tratamiento'
data={
  'accion':'actualizarTipoTratamiento',
  'idTipoTratamiento': None,
  'tipoTratamiento': None,
}
print postear(data,url)
print

url='Tratamiento'
data={
  'accion':'actualizar',
  'dosis': None,
  'idTipoTratamiento': None,
  'intervaloTiempo': None,
  'fechaInicio': None,
  'idViaSuministro': None,
  'idTratamiento': None,
  'duracionTratamiento': None,
}
print postear(data,url)
print

File: BackEnd/ajax/Tratamiento.js
$.ajax({
  url:SITIO_URL+'/services/Tratamiento.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'crear',
    'dosis': null,
    'intervaloTiempo': null,
    'fechaInicio': null,
    'duracionTratamiento': null,
    'idTipoTratamiento': null,
    'idViaSuministro': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Tratamiento.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'agregarTipoTratamiento',
    'tipoTratamiento': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Tratamiento.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorPaciente',
    'idPaciente': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Tratamiento.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarTipoTratamiento',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Tratamiento.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'actualizarReceta',
    'idTratamiento': null,
    'idConsulta': null,
    'idMedico': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Tratamiento.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'agregarViaSuministro',
    'viaSuministro': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Tratamiento.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarViaSuministro',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Tratamiento.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'recetar',
    'idTratamiento': null,
    'idConsulta': null,
    'idMedico': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Tratamiento.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'actualizarViaSuministro',
    'idViaSuministro': null,
    'viaSuministro': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Tratamiento.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'actualizarTipoTratamiento',
    'tipoTratamiento': null,
    'idTipoTratamiento': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Tratamiento.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'actualizar',
    'idTratamiento': null,
    'dosis': null,
    'intervaloTiempo': null,
    'fechaInicio': null,
    'duracionTratamiento': null,
    'idTipoTratamiento': null,
    'idViaSuministro': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


